src/middlewares/userValidator.js

```diff
@@ -38,6 +39,17 @@ const userValidator = {
     isValidEmail(),
     validatorError
   ],
+  validateCreateUser: [
+    isValidEmail(),
+    isValidName('firstName'),
+    isValidName('lastName'),
+    isValidName('roleName'),
```

Comment:
I think `roleName` shouldn't use the `isValidName` validator. This will allow any random name to be entered as roleName. Instead, the validator for role name should check if it's among the predefined roles.

Reply:
You didn't implement this feedback...

Reply:
I fixed this in the next PR. Brokedown the task into two:  create and read/update and delete. I didn't do it here to avoid duplication.

Reply:
I will let this slide for now but next time, implement the feedback on the request PR